test/integration/default/data_bags/x86_service_passwords/openstack-orchestration.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "openstack-orchestration",
+  "openstack-orchestration": {
+    "encrypted_data": "WLrdEYvl1Oj+vEhXu/BcTP2KkuI3I28Fu1i3zNwjlzld9rsajTB4WX9Xd/Xh\nzvkI\n",
+    "hmac": "GRMw/UWGn1rm5gIFMu5HEjLauQ0Wr4Zlezy2DAz4CNc=\n",
+    "iv": "GvU4RmsHJbjnEzrWFzMRow==\n",
+    "version": 2,
+    "cipher": "aes-256-cbc"
+  }
+}
```

Comment:
github complaining about a newline?

Reply:
I can't control this as it's created via knife. I would ignore this.

Reply:
How did the other data_bag get a newline? hmm...